### PR TITLE
update french translation

### PR DESCRIPTION
**Pull request by Ant1r**

### CODE REVIEW

File: po/fr.po [hunk 1]
```diff
@@ -30,25 +30,25 @@ msgid "-----------output-range:-----------"
 msgstr "---------échelle_de_sortie:--------"
 
 msgid "------selectable_dimensions(pix):------"
-msgstr "-------dimensions_a_choisir(pix):------"
+msgstr "-------dimensions_sélectionnable(pix):------"
```
Comment (danomatika) [inline po/fr.po]: This needs to be the same width as the string above. Easiest option is to remove some of the "-" characters.

File: po/fr.po [hunk 2]
```diff
@@ -6,16 +6,16 @@ msgstr ""
...
+"Last-Translator: Cyrille Henry / Antoine Rousseau / Antoine Villeret / Baptiste Chatel\n"
 "Language-Team: \n"
 "Language: fr\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 
 msgid "-------dimensions(digits)(pix):-------"
-msgstr "-------dimensions(digits)(pix):-------"
+msgstr "-------dimensions(chiffres)(pix):-------"
```
Comment (danomatika) [inline po/fr.po]: This needs to be the same width as the string above.

**Comment by danomatika** [comment]
Why were spaces added before semicolons? Is this a language convention I'm unaware of? Otherwise, I think the semicolons should be adjacent to the text as this affects the label sizing.

**Comment by Ant1r** [comment]
> Why were spaces added before semicolons? Is this a language convention?
Yes, french typography requires a space before and after any "double" sign, like: : ! ? ;

Anyway I'll fix other things before re-submitting.

**Comment by Ant1r** [comment]
I simplified this PR and reopen as #250, directed to master.

**Pull request closed without merging**